### DISCUSSION TITLE
Bloomberg has given $10000 to MkDocs

### DISCUSSION THREAD

**Discussion by oprypin** [mkdocs/mkdocs#3369] [Announcements]
https://twitter.com/TechAtBloomberg/status/1697678209326731291

The company Bloomberg has given a grant to the project MkDocs. This is part of [the 2nd round of grants](https://www.bloomberg.com/company/stories/open-source-bloomberg-1h-2023/) from [Bloomberg FOSS Contributor Fund](https://www.bloomberg.com/company/stories/bloomberg-ospo-launches-foss-contributor-fund/) ⁠— and the amount is, as with all other such grants, USD $10000.

To receive the grant, I have created the page https://opencollective.com/mkdocs. (And that's really the only purpose of this page at the moment). I invited core contributors @ultrabug and @pawamoy and reached out to past core maintainers in case they were interested in the opportunity. From there, we settled on the principle that the money will be sent out to individuals, and the amounts sent will be publicly visible. MkDocs has no expenses to speak of, so there’s no plan to spend the money on anything other than rewarding individuals for their volunteer work.

Thank you very much, Bloomberg!

This was a big surprise. It happened just as I myself was starting to accept donations for MkDocs, and blew those donations out of the water.

The grant was open-ended. And we recognize that the experience of using MkDocs is really not all about the core MkDocs project; the themes and plugins play just as big of a role, if not more so in many cases.
So I reached out to a few authors of projects related to MkDocs to see if they would like to be rewarded for their work, as a one-time payment.

For now we are still figuring out the OpenCollective system, and only small payments have been confirmed so far, to try it out. But soon we will assign a big portion of the money to myself and core contributors, as well as do a bit of outreach. The rest of the money will stay on the OpenCollective account for now. Later we'll better see how this is going and what makes sense ⁠— whether to add more recipients or perhaps send more to the same recipients.

**Comment by pawamoy** [comment]
My warm thanks to Bloomberg and Bloomberg employees as well! As a full time open source developer myself, this kind of funding is super appreciated :slightly_smiling_face:

**Comment by Andre601** [comment]
Wow... 10k. I wish I was that lucky. 😅
Good job. You earned it for sure.

**Comment by ultrabug** [comment]
Warm thanks as well here to Bloomberg, I feel humbled by this amazing and unexpected grant :bow: .

**Comment by dazinator** [comment]
It's great to see open source projects that aren't Microsoft or Google owned, getting some tangible appreciation.

**Comment by trel** [comment]
congratulations.

**Comment by facelessuser** [comment]
You all are doing great work, and I'm happy to see an organization like Bloomberg recognizing all the work you do!

**Comment by squidfunk** [comment]
More than deserved! MkDocs is powering a significant portion of the world's documentation. Congrats to the core team! 🚀

**Comment by phil65** [comment]
Really cool, congratulations!

**Comment by armandino** [comment]
Congratulations! MkDocs is a great project. Happy to see it being rewarded! :heart:

**Comment by lovelydinosaur** [comment]
Wonderful news indeed. Great work team. 💚
